packages/ffe-form/less/phone-number.less

```diff
@@ -3,6 +3,10 @@
         @media (min-width: @breakpoint-sm) {
             display: flex;
         }
+        & > * {
```

Comment:
Kunne man ha lagt `margin-bottom` på  `.ffe-phone-number__input-group` istedenfor alle elementer som ligger inni?   Sånn jeg tolker eksempelet på component-overview, så havner feilmeldingen under den klassen.
Trenger du også å sette` margin-top: 0` her?

Reply:
> Synes alltid det er litt skummelt å style "alle" elementer inni en input-group.

Enig i det. Men akkurat denne er nøstet spesifikt i `.ffe-phone-number__input-group`, så i praksis vil det ikke gjelde så altfor mange elementer.

Reply:
Nei, det funker i dette tilfelle, men det er litt dårlig praksis. 
Ingenting som stopper merging, men greit å være klar over og at det finnes alternativer som f.eks å bruke :has for å sjekke på at child-element har en bestemt klasse  `& > *:has(.ffe-field-message)` for å gjøre det litt mer "isolert"

Reply:
Gjorde det du skrev først, men det fungerte ikke på mobil! Helt enig at det er skummelt. Er det som er gjort for input group, det bare når ikke phone-number input group fordi det er et ekstra lag innenfor. 
![image](https://github.com/SpareBank1/designsystem/assets/155720728/2ed7344f-5079-4f36-ad36-784905ee8591)

Prøvde også å legge til margin-top på ffe-field-message (bra vi tenker likt hehe), men den blir overstyrt av ` > * { margin: 0; }` fra input group. 
![image](https://github.com/SpareBank1/designsystem/assets/155720728/14deb063-c3f7-4f5b-b6fd-99714a85d444)